packages/core/src/components/axes/ruler.ts

```diff
@@ -0,0 +1,206 @@
+// Internal Imports
+import { Component } from "../component";
+import { DOMUtils } from "../../services";
+import textWidth from "text-width";
+import { scaleLinear } from "d3-scale";
+import { Selection, BaseType } from "d3-selection";
+import { isEqual } from "lodash-es";
```

Comment:
1. The 3 imports above aren't internal so they should be part of the other block
2. `d3-selection` was already imported below, we should add those modules to the import statement below